app/src/main/java/co/tinode/tindroid/MessagesFragment.java

```diff
@@ -690,6 +702,14 @@ private void sendText() {
         }
     }
 
+    // Send video as a link
+    private static Drafty draftyVideo(String mimeType, byte[] bits, int width, int height, String fname,
```

Comment:
This method appears to be unused. Why do you need it?

Reply:
For video sharing feature

Reply:
Yes, I understand that it's for video, but it's currently unused. It also has nothing to do with mentions that you otherwise work on. Please keep it simple:

* All code related to a feature should be in one PR. The PR should compile and work as expected.
* Code not related to the feature should not be in the PR. If it's not related to the feature keep it out. Send it separately in another PR.